src/alignment.js

```diff
@@ -121,7 +121,7 @@ var alignWithDOM = function(nodeName, key, statics) {
 /**
  * Clears out any unvisited Nodes, as the corresponding virtual element
  * functions were never called for them.
- * @param {!Element} node
+ * @param {!Node} node
```

Comment:
This should always be an Element, you can't open a `TextNode` (we use `nextSibling`).